Formula/rotz.rb

```diff
@@ -0,0 +1,22 @@
+class Rotz < Formula
+  desc "Fully cross platform dotfile manager and dev environment bootstrapper"
+  homepage "https://volllly.github.io/rotz/"
+  url "https://github.com/volllly/rotz/archive/refs/tags/v0.9.3.tar.gz"
```

Comment:
The version should be declared in a `version` stanza an used as a variable here. Perhaps an even better option would be to use `livecheck` then we would not need to update the formula manually on each update:

https://docs.brew.sh/Cask-Cookbook#stanza-livecheck

Reply:
It definitely should not because this is a formula and not a cask

Reply:
Yup, I got confused between the behavior of casks and formulas. Sorry about that.